src/main/resources/themes/default/style.css

```diff
@@ -581,6 +581,28 @@
   -fx-background-color: linear-gradient(to left, transparent, -fx-accent);
 }
 
+/***************** Large Map Preview *****************/
+.map-preview-large{
```

Comment:
Alright, but that doesn't affect the importance of seemingly unimportant things :-) If people are trained to be careful about even "unimportant" things they'll (hopefully) be careful about important things.